enterprise/backend/src/metabase_enterprise/serialization/v2/seed_entity_ids.clj

```diff
@@ -0,0 +1,114 @@
+(ns metabase-enterprise.serialization.v2.seed-entity-ids
+  (:require
+   [clojure.string :as str]
+   [clojure.tools.logging :as log]
+   [metabase.db :as mdb]
+   [metabase.db.connection :as mdb.connection]
+   [metabase.models]
+   [metabase.models.dispatch :as models.dispatch]
+   [metabase.models.serialization.hash :as serdes.hash]
+   [metabase.util :as u]
+   [metabase.util.i18n :refer [trs]]
+   [toucan.db :as db]
+   [toucan.models :as models]))
+
+(set! *warn-on-reflection* true)
```

Comment:
Is this leftover debugging code? I'm not sure the style guide on this.

Reply:
It's actually the preferred way of doing stuff these days... it only affects the current namespace. There's a Kondo linter for it now: https://github.com/clj-kondo/clj-kondo/blob/master/doc/linters.md#warn-on-reflection

It's something we should actually do for all of our namespaces using Java interop but it's going to be a lot of busy work fixing everything so I haven't gotten around to it. I do intend to enable it at some point in the future tho